python/ql/src/experimental/semmle/python/frameworks/Stdlib.qll

```diff
@@ -111,14 +111,26 @@ private module Stdlib {
   }
 
   /**
-   * A call to `os.popen`
+   * A call to any of the `os.popen*` functions
    * See https://docs.python.org/3/library/os.html#os.popen
+   *
+   * Note that in Python 2, there are also `popen2`, `popen3`, and `popen4` functions.
+   * Although deprecated since version 2.6, they still work in 2.7.
+   * See https://docs.python.org/2.7/library/os.html#os.popen2
    */
   private class OsPopenCall extends SystemCommandExecution::Range {
-    OsPopenCall() { this.asCfgNode().(CallNode).getFunction() = os_attr("popen").asCfgNode() }
+    string name;
+
+    OsPopenCall() {
```

Comment:
This one can also be cleaned up by extending `CfgNode`, I think.

Reply:
Done :+1: